tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js

```diff
@@ -0,0 +1,529 @@
+      await this.pageObjects.boBasePage.goToSubMenu(
+        this.pageObjects.boBasePage.customersParentLink,
+        this.pageObjects.boBasePage.customersLink,
+      );
+      const pageTitle = await this.pageObjects.customersPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.customersPage.pageTitle);
+    });
+
+    it('should filter list by email', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterToViewCustomer', baseContext);
+      await this.pageObjects.customersPage.resetFilter();
+      await this.pageObjects.customersPage.filterCustomers(
+        'input',
+        'email',
+        createCustomerData.email,
+      );
+      const textEmail = await this.pageObjects.customersPage.getTextColumnFromTableCustomers(1, 'email');
+      await expect(textEmail).to.contains(createCustomerData.email);
+    });
+
+    it('should click on view customer', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToViewCustomerPageAfterCreateOrder', baseContext);
+      await this.pageObjects.customersPage.goToViewCustomerPage(1);
+      const pageTitle = await this.pageObjects.viewCustomerPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.viewCustomerPage.pageTitle);
+    });
+
+    it('should check personal information title', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkPersonalInformationTitle2', baseContext);
+      const cardHeaderText = await this.pageObjects.viewCustomerPage.getPersonalInformationTitle();
+      await expect(cardHeaderText).to.contains(createCustomerData.firstName);
+      await expect(cardHeaderText).to.contains(createCustomerData.lastName);
+      await expect(cardHeaderText).to.contains(createCustomerData.email);
+    });
+
+    it('should check customer personal information', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkCreatedCustomerInfo2', baseContext);
+      const cardHeaderText = await this.pageObjects.viewCustomerPage.getTextFromElement('Personal information');
+      await expect(cardHeaderText).to.contains(createCustomerData.socialTitle);
+      await expect(cardHeaderText).to.contains(`birth date: ${customerBirthDate}`);
+      await expect(cardHeaderText).to.contains(todayDate);
+      await expect(cardHeaderText).to.contains(Languages.english.name);
+      await expect(cardHeaderText).to.contains('Newsletter');
+      await expect(cardHeaderText).to.contains('Partner offers');
+      await expect(cardHeaderText).to.contains('Active');
+    });
+
+    const tests = [
+      {args: {blockName: 'Orders', number: 1}},
+      {args: {blockName: 'Carts', number: 1}},
+      {args: {blockName: 'Viewed products', number: 1}},
+      {args: {blockName: 'Messages', number: 1}},
+      {args: {blockName: 'Vouchers', number: 0}},
+      // {args: {blockName: 'Last emails', number: 2}},
```

Comment:
You can remove this line

Reply:
It was a bug :) now it fixed I will delete the //